src/nvim/eval.c

```diff
@@ -15239,9 +15242,20 @@ static void set_qf_ll_list(win_T *wp, typval_T *list_arg, typval_T *action_arg,
         action = *act;
     }
 
-    if (l != NULL && set_errorlist(wp, l, action,
-            (char_u *)(wp == NULL ? "setqflist()" : "setloclist()")) == OK)
+    if (title_arg->v_type == VAR_STRING) {
```

Comment:
What's the proper way of catching errors? `helpers.exc_exec('call setqflist([""], "r", {})')` doesn't seem to return an error message.